src/main/java/core/basesyntax/handler/ConsoleHandler.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler implements Handler {
+    @Inject
+    private BetDao betDao;
+    @Dao
+    private FruitDao fruitDao;
+
+    @Override
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            Bet bet = null;
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                String[] parseString = input.split(" ");
+                int value = Integer.parseInt(parseString[0]);
+                double risk = Integer.parseInt(parseString[1]);
+
+                if (risk < 0 || value < 0) {
+                    System.out.println("Value is negative");
+                    continue;
+                }
+                bet = new Bet(value, risk);
+                System.out.println(bet != null ? bet.toString() : "");
```

Comment:
```suggestion
                System.out.println(bet);
```
Redundant check, condition 'bet != null' is always 'true'

Reply:
done